extension/src/test/suite/registryProvider.test.ts

```diff
@@ -212,6 +231,18 @@ const USER_REGISTRY_CONFIG = {
     ],
 };
 
+const USER_REGISTRY_CONFIG_CHNL = {
+    registries: [
+        {
+            name: 'User Registry',
+            registry: USER_REGISTRY_URL,
+        },
+    ],
+    channels: {
+        test: 'insiders',
```

Comment:
Note that if we fix channels to use the full extension ID, this needs to be updated.